workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-cron/src/main/java/com/liferay/partner/PartnerBaseController.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR
+ * LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.partner;
+
+import com.liferay.petra.string.StringBundler;
+
+import java.net.URI;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import java.util.function.Function;
+
+import org.json.JSONObject;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.UriBuilder;
+
+/**
+ * @author Elvison Victor
+ */
+public abstract class PartnerBaseController {
```

Comment:
public abstract class PartnerBaseController  to public abstract class BasePartnerController {